src/kirby/components/floating-action-button/floating-action-button.component.html

```diff
@@ -1,3 +1,3 @@
-<ion-fab-button class="fab {{ showShadow ? '' : 'no-shadow' }}" [disabled]="disabled">
+<ion-fab-button class="fab {{ !showShadow || disabled ? 'no-shadow' : '' }} {{ disabled ? 'disabled' : '' }}" [disabled]="disabled">
```

Comment:
Could be simplified, by using a `HostBinding` in the component instead:
```ts
@HostBinding('class.has-shadow')
@Input() showShadow?: boolean = true;
```

And then in the scss:
```css
ion-fab-button,
ion-fab-button('[disabled]') {
  --box-shadow: 0;
}
:host('.has-shadow')  ion-fab-button:not('[disabled]') {
 --box-shadow: 0 10px 20px -10px #{get-color('kirby-primary-shade')};
}
```

Reply:
Could't get it to work with HostBinding  but it was simplified in other way